### PR TITLE
Add Sqlectron.app v1.1.1

### PR DESCRIPTION
**Pull request by maxclaus**
Requested by issue sqlectron/sqlectron-gui#73

### CODE REVIEW

File: Casks/sqlectron.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+cask :v1 => 'sqlectron' do
```
Comment (jawshooah) [inline Casks/sqlectron.rb]: Please change to `cask 'sqlectron' do`

File: Casks/sqlectron.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  version '1.1.1'
+  sha256 '9f604513342e4ce3f4bfedeca58f9d9d7b86f36640a0f2e603ae741b7cd6e1bc'
+
+  # github.com is the official download host per the vendor homepage
```
Comment (jawshooah) [inline Casks/sqlectron.rb]: This is unnecessary, since the homepage is hosted on `github.io`.

**Comment by maxcnunes** [comment]
Applied the suggested changes.

I have a quick question. I realized the path `/opt/homebrew-cask/Caskroom/sqlectron/1.1.1/` keeps the `Sqlectron.app` and `osx/Sqlectron.dmg`. Is possible to remove the `dmg` file after the installation is completed? Just to not consume useless space in disk.

**Comment by jawshooah** [comment]
That will likely be resolved as a result of #13201. For now, it's not a big deal. Internal disks are big, and most people can spare 50MB.

**Comment by jawshooah** [comment]
Thank you for the contribution. It was merged directly as 4b5b547c974996e58f55aefad9f2d5472c659efb to keep commit history cleaner. Your contribution is still credited to you.

**Comment by maxcnunes** [comment]
Awesome. Thanks!

**Pull request closed without merging**